stacker/actions/build.py

```diff
@@ -72,6 +72,20 @@ def should_submit(stack):
     return False
 
 
+def should_ensure_cfn_bucket(outline, dump):
+    """Test whether access to the cloudformation template bucket is required
+
+    Args:
+        outline (bool): The outline action.
+        dump (bool): The dump action.
+
+    Returns:
+        bool: If access to CF bucket is needed, return True.
+
+    """
+    return outline is False and dump is False
```

Comment:
may need to consider package sources as well.  if user sets up package sources then I think stacker needs to run in online mode.